src/poprox_recommender/domain/article.py

```diff
@@ -1,12 +1,31 @@
 from datetime import datetime, timezone
+from typing import List, Optional
 from uuid import UUID
 
 from pydantic import BaseModel
 
 
+class Entity(BaseModel):
+    entity_id: Optional[UUID] = None
```

Comment:
Curious why `entity_id` is Optional

Reply:
We could generate the UUIDs ourselves in Python if there's a reason to prefer that, but I'm most confident that Postgres will generate Postgres-compatible UUIDs